requirements.txt

```diff
@@ -1,7 +1,8 @@
 docker < 4.3.0
 docker-squash>=1.0.7
 dockerfile-parse>=0.0.13
-flatpak-module-tools >= 0.11
+flatpak-module-tools >= 0.11,<0.13;python_version<"3.9"
+flatpak-module-tools >= 0.11;python_version>="3.9"
```

Comment:
shouldn't this be : 
flatpak-module-tools >= 0.13;python_version>="3.9"

Reply:
not necessary, lower version should still work under py3.9